src/Education/EducationPackage.cs

```diff
@@ -32,11 +34,24 @@ namespace SonarLint.VisualStudio.Education
     [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [Guid("7ef4a2de-4035-48c3-b273-4195d0f1186b")]
     [ProvideMenuResource("Menus.ctmenu", 1)]
+    [ProvideToolWindow(typeof(RuleDescriptionToolWindow), MultiInstances = false, Style = VsDockStyle.Tabbed, Window = ToolWindowGuids.SolutionExplorer, Width = 325, Height = 400)]
```

Comment:
Copied from issueviz.  We might adjust things like size and style but I figured for right now its fine to be this way.